src/app/pages/dashboard/dashboard.page.html

```diff
@@ -0,0 +1,45 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <button class="logout-button" (click)="openLogoutModal()" *ngIf="isLogoutEnabled()">
+        <span>Logout</span>
+        <h6 class="version-number">Version {{pageState.appVersion$ | async}}</h6>
+      </button>
+      <h6 *ngIf="!isLogoutEnabled()" class="version-number">Version {{pageState.appVersion$
+        | async}}</h6>
+      <input type="hidden" name="employeeId" id="employeeId" value="{{employeeId}}" />
+    </ion-buttons>
+    <ion-title>My dashboard</ion-title>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content [fullscreen]="true">
+
+  <div>
+    <profile-header [employeeId]="employeeId" [name]="pageState.employeeName$ | async" [role]="role"></profile-header>
+    <hr/>
+    <h4 class="date">{{todaysDateFormatted}}</h4>
+    <ion-grid class="dashboard-grid">
+      <ion-row>
+        <ion-col>
+          <go-to-journal-card></go-to-journal-card> <!-- *ngIf="!showDelegatedExaminerRekey()" -->
```

Comment:
In these cases I would prefer to see the functions like `showDelegatedExaminerRekye()` in the Angular class just it would return a default value `true` and maybe exposed a TODO comment or `console.log` saying that it's not yet implemented.

Reply:
Good point, this was a quick job to stop the errors showing. I will make the necessary changes now